{"uid":"178037d826fdfe0d","name":"should propagate synchronous error","fullName":"test/Publish.test.ts#Publish Mechanism Error propagation Non-async errors without scope should propagate synchronous error","historyId":"11e167fdf6057d086dca39e967bacf3d:d41d8cd98f00b204e9800998ecf8427e","time":{"start":1743018241520,"stop":1743018241546,"duration":26},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"framework","value":"vitest"},{"name":"language","value":"javascript"},{"name":"parentSuite","value":"Publish Mechanism"},{"name":"suite","value":"Error propagation"},{"name":"subSuite","value":"Non-async errors without scope"},{"name":"host","value":"fv-az1938-282"},{"name":"thread","value":"vitest-worker-1"},{"name":"package","value":"test.Publish.test.ts"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":[]},"source":"178037d826fdfe0d.json","parameterValues":[]}